driver/src/main/java/com/code_intelligence/jazzer/driver/FuzzTargetFinder.java

```diff
@@ -209,4 +213,39 @@ private static Optional<Method> targetPublicStaticMethod(
       return Optional.empty();
     }
   }
+
+  // Returns a list of all methods annotated with @FuzzTest without requiring @FuzzTest to be on the
```

Comment:
In which case is the annotation missing on the classpath?

Reply:
For example, when users package their tests into a JAR in Maven and pass it to the Jazzer CLI via `--cp`, they would be surprised to find that their fuzz tests aren't executing - they would have to list our JUnit integration as well as all its JUnit dependencies as well. This would make it very difficult to execute a `@FuzzTest`-based fuzz test on the command-line if not for this change.